wcfsetup/install/files/lib/data/blacklist/entry/BlacklistEntryAction.class.php

```diff
@@ -75,7 +76,7 @@ public function import() {
 					foreach ($data[$type] as $hash => $occurrences) {
 						$statement->execute([
 							$type,
-							hex2bin($hash),
+							Hex::decode($hash),
```

Comment:
Not here, please. This is not security critical usage and also relies on unverified data.